src/GtkUtilities.jl

```diff
@@ -44,7 +44,7 @@ include("panzoom.jl")
 using .PanZoom
 import .PanZoom: interior, fullview   # for extensions
 
-function Base.copy!{C<:Colorant}(ctx::CairoContext, img::AbstractArray{C})
+function Base.copy!{C<:Union{Colorant,Number}}(ctx::CairoContext, img::AbstractArray{C})
```

Comment:
That's exactly the case here:
- AbstractArray and `copy!` are defined in one package, which happens to be called Base
- Canvas is defined in another package, called Gtk

This package depends on both, and adds functionality that bridges the gap between these two "packages." It doesn't override behavior that has a definition anywhere else; it's a `MethodError` if you try that without loading this package, just the same as it would be if you tried `x::NumberType1 + y::NumberType2` without loading the package that glues them together (well, that would be a "no promotion defined" error, but it's conceptually the same thing).